01-creating-a-transaction/index.js

```diff
@@ -7,29 +7,33 @@ const sdk = new SDK({ url: 'https://eos.greymass.com' }) // loaded with default
 
 async function main() {
     const session = await sdk.login('myapp') // login also restores existing session unlike anchor-link, can have option for forcing re-login
-    
+
     // Loads contract by name
     const contract = await sdk.getContract('rewards.gm')
-    
+
     // Work with table data
-    const usersTable = await contract.getTable('users')
-    const row = await usersTable.getByPrimary('teamgreymass')
-    const amount = row.getColumn('amount')
-    
+    const usersTable = contract.getTable('users')
+    const rows = await usersTable.getByPrimary('teamgreymass')
+    const amount = rows[0].getColumn('amount')
+
     // This could have a strongly typed interface as well
-    class MyRow extends Struct {
-        @Struct.field('uint64') declare amount: UInt64
-    }
-    const usersTable = await contract.getTable('users', MyRow)
-    const row = await usersTable.getByPrimary('teamgreymass')
-    const amount = row.amount
-    
+    // class MyRow extends Struct {
+    //     @Struct.field('uint64') declare amount: UInt64
+    // }
+    // const usersTable = await contract.getTable('users', MyRow)
+    // const row = await usersTable.getByPrimary('teamgreymass')
+    // const amount = row.amount
+
```

Comment:
Commented this out because I wanted the example to look function, and this was duplicating the logic twice.  It shows the alternative path without making it look like it's required to complete the code sample.